src/drivers/cmakeServerDriver.ts

```diff
@@ -154,7 +154,7 @@ export class CMakeServerDriver extends CMakeDriver {
         })();
     }
 
-    protected async doConfigure(args: string[], consumer?: proc.OutputConsumer, showCommandOnly?: boolean, configurePreset?: ConfigurePreset | null) {
+    protected async doConfigure(args: string[], consumer?: proc.OutputConsumer, showCommandOnly?: boolean, configurePreset?: ConfigurePreset | null, _options?: proc.ExecutionOptions) {
         await this._clientChangeInProgress;
         const cl = await this.getClient();
```

Comment:
`this.getClient()` calls `this._startNewClient()`.  If you save the `options?.environment`, you can pass the env to `this.getConfigureEnvironment()` when the server is started.

Reply:
We talked about this and decided not to support CMake server.